DependencyInjection/Configuration.php

```diff
@@ -179,9 +179,37 @@ private function addErrorHandlerSection(NodeBuilder $nodeBuilder)
     {
         $nodeBuilder
             ->arrayNode('error')
+                ->fixXmlConfig('template')
+                ->fixXmlConfig('excluded_path')
+                ->fixXmlConfig('exclusion_rule')
                 ->children()
                     ->scalarNode('enable_parent_provider')->defaultValue(false)->end()
                     ->scalarNode('enable_sibling_provider')->defaultValue(false)->end()
+                    ->arrayNode('templates')
+                        ->useAttributeAsKey('format')
+                        ->requiresAtLeastOneElement()
+                        ->defaultValue(array(
+                            'html' => 'CmfSeoBundle:Exception:exception.html.twig',
+                            'xml'  => 'TwigBundle:Exception:error.xml.twig',
+                            'json' => 'TwigBundle:Exception:error.json.twig',
+                        ))
+                        ->prototype('scalar')->end()
+                    ->end()
+                    ->arrayNode('exclusion_rules')
+                        ->info('Rules to exclude error handling from specific matches.')
+                        ->prototype('array')
+                            ->children()
+                                ->arrayNode('match')
+                                    ->children()
+                                        ->scalarNode('path')->defaultNull()->info('Path to exclude')->end()
+                                        ->scalarNode('host')->defaultNull()->info('Host to exclude')->end()
+                                        ->scalarNode('method')->defaultNull()->info('method to exclude')->end()
```

Comment:
Users don't know the implementation behind it. I would prefer to have clear settings, thinking about what is usefull and what not.

E.g., it might be usefull to have a new content type option here. To avoid JSON requests to get the special error page for instance.